websites/A/AtCoder/dist/metadata.json

```diff
@@ -10,9 +10,9 @@
 		"vi_VN": "AtCoder là trang tổ chức cuộc thi lập trình dành cho mọi người từ mới bắt đầu đến các chuyên gia. Chúng tôi chủ trì các cuộc thi lập trình hàng tuần."
 	},
 	"url": "atcoder.jp",
-	"version": "1.0.9",
-	"logo": "https://i.imgur.com/sljarC4.png",
-	"thumbnail": "https://i.imgur.com/zuvLy06.png",
+	"version": "1.0.10",
+	"logo": "https://i.imgur.com/CpbLpxS.png",
+	"thumbnail": "https://i.imgur.com/0yqTD48.png",
```

Comment:
is there no better thumbnail for this website? It's fine if there isn't but it would be nice to get something that doesn't show dates, which this one does